src/components/SolicitudCard.tsx

```diff
@@ -79,7 +79,8 @@ export default function SolicitudCard({
               <div className="flex items-start gap-2">
                 <MapPinned className="h-4 w-4 text-gray-500 flex-shrink-0 mt-1" />
                 <span className="break-words">
-                  <span className="font-semibold">Pueblo:</span> {towns[caso.town_id - 1]?.name || ''}
+                  <span className="font-semibold">Pueblo:</span>{' '}
+                  {towns.filter((town) => town.id === caso.town_id)[0]?.name || ''}
```

Comment:
You can use a find instead of filter and getting the first element

Reply:
done